src/db/pgsql/Schema.php

```diff
@@ -164,6 +167,17 @@ public function getDefaultRestoreCommand(): string
             ->addArg('--username=', '{user}')
             ->addArg('--no-password');
 
+        if ($this->usePgRestore()) {
+            $command
+                ->addArg('--clean')
+                ->addArg('--if-exists')
+                ->addArg('--no-owner')
+                ->addArg('--no-acl')
+                ->addArg('--schema=', '{schema}')
+                ->addArg('--single-transaction')
+                ->addArg('{file}');
+        }
+
```

Comment:
When using pg_restore, these args need to be on the restore, regardless if the backup was created with them.